### PR TITLE
API1800 support for Interconnects

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
API1800 support for Interconnects

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
  - [ ] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [ ] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -83,10 +83,10 @@
...
-|<sub>/rest/interconnects/{id}/resetportprotection</sub>                                | PUT      |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |
-|<sub>/rest/interconnects/{id}/update-port</sub>                                        | PUT    | :white_check_mark:  | :white_check_mark:   |  :white_check_mark:  | :white_check_mark:  |:white_check_mark:   | :white_check_mark: | :white_check_mark:  | :white_check_mark:  |
+|<sub>/rest/interconnects/{id}</sub>                                                    | PATCH    | :white_check_mark:  | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |:white_check_mark:   | :white_check_mark: | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  |
+|<sub>/rest/interconnects/{id}/configuration</sub>                                      | PUT    | :white_check_mark:  | :white_check_mark:   | :white_check_mark:  | :white_check_mark:  |:white_check_mark:   | :white_check_mark: | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  |
```
Comment (VenkateshRavula) [inline endpoints-support.md]: Add only one field, remove other one.

File: endpoints-support.md [hunk 2]
```diff
@@ -83,10 +83,10 @@
...
+|<sub>/rest/interconnects/{id}/resetportprotection</sub>                                | PUT      |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  | :white_check_mark:  |
+|<sub>/rest/interconnects/{id}/update-port</sub>                                        | PUT    | :white_check_mark:  | :white_check_mark:   |  :white_check_mark:  | :white_check_mark:  |:white_check_mark:   | :white_check_mark: | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  |
```
Comment (VenkateshRavula) [inline endpoints-support.md]: Add only one field, remove other one.

Comment (yuvirani) [reply endpoints-support.md]: Done

File: endpoints-support.md (hunk 1 above)
Comment (yuvirani) [reply endpoints-support.md]: Done

**Review by VenkateshRavula — APPROVED** [review]

**Review by nabhajit-ray — APPROVED** [review]

**Pull request merged**